### PR TITLE
Upgrade to 3.0.1

### PR DESCRIPTION
**Pull request by alexjbest**
rebased version of #9

Due to incompatibility of symmetrica with windows c headers (defining min and max which symmetrica also wants to define) we must compile in compatibility mode, so we add the necessary flag (/Za) to msvc to do this in cmakelists.

c.f https://gitlab.com/sagemath/symmetrica/-/issues/6, https://gitlab.com/sagemath/symmetrica/-/blob/master/README.md#resolving-minmax-name-conflicts

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by alexjbest** [comment]
@conda-forge-admin please rerender

**Comment by alexjbest** [comment]
@conda-forge-admin please rerender

**Comment by alexjbest** [comment]
@isuruf this is now ready for review

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,7 +35,7 @@ test:
 about:
   home: http://www.algorithm.uni-bayreuth.de/en/research/SYMMETRICA/index.html
   license: ISC
-  license_file: {{ RECIPE_DIR }}/LICENSE
+  license_file: LICENSE
   summary: |
     Symmetrica is a collection of routines to handle several topics including
     ordinary representation theory of the symmetric group and related groups
```
Comment (saraedum) [inline recipe/meta.yaml:41]: @alexjbest could you add the two of us as maintainers here before this gets merged?

Comment (alexjbest) [reply recipe/meta.yaml:41]: @saraedum Done!

**Comment by alexjbest** [comment]
@conda-forge-admin please rerender

**Review by saraedum — APPROVED** [review]

**Comment by saraedum** [comment]
@isuruf, I think that this one is good to go.

**Comment by isuruf** [comment]
Thanks

**Pull request merged**